Tests/Shared.Specs/Shared.Specs.projitems

```diff
@@ -9,81 +9,79 @@
     <Compile Include="$(MSBuildThisFileDirectory)ExtensibilitySpecs.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)FindAssembly.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)FluentDateTimeSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)FormatterSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)GenericCollectionAssertionOfStringSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)GenericCollectionAssertionsSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)GenericDictionaryAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)GuidAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)MethodBaseAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)MethodInfoAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)MethodInfoSelectorAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)MethodInfoSelectorSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)NullableBooleanAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)NullableGuidAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)NullableNumericAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)NullableSimpleTimeSpanAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)NumericAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ObjectAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ObjectCastingSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ObjectExtensionsSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)PropertyInfoAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)PropertyInfoSelectorAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)PropertyInfoSelectorSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)ReferenceTypeAssertionsSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)SimpleTimeSpanAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)StringAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TaskAssertionSpecs.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)TaskOfTAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Numeric\ComparableSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Numeric\NullableNumericAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Numeric\NumericAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\BooleanAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\DateTimeAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\DateTimeOffsetAssertionSpecs.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Primitives\DateTimeOffsetValueFormatterSpecs.cs" />
```

Comment:
Move `DateTimeOffsetValueFormatterSpecs` and `TimeSpanFormatterSpecs` be moved to `Formatting/`?